packages/builder/src/components/start/AppRow.svelte

```diff
@@ -18,14 +18,26 @@
   export let deleteApp
   export let unpublishApp
   export let releaseLock
+  export let editIcon
+
+  $: color =
+    $apps.find(filtered_app => app?.appId === filtered_app.appId)?.icon
```

Comment:
Does the passed in `app` prop not contain this info already in `app.icon`? Or is there a reason that we need to find the app definition from the list again?